cmd/mimir/config-descriptor.json

```diff
@@ -4622,6 +4622,17 @@
           ],
           "fieldValue": null,
           "fieldDefaultValue": null
+        },
+        {
+          "kind": "field",
+          "name": "response_streaming_enabled",
+          "required": false,
+          "desc": "Enables streaming of responses from querier to query-frontend for responses that request it. This is an experimental feature and may be removed in the future.",
```

Comment:
> for responses that request it

This is confusing to me given the relationship of requests and responses. 
Would you mind explaining it to me a little more to help me understand what this means more precisely?

Reply:
I see how this can be confusing, let me try to explain, maybe you can help me word it better:

When the querier is done computing the response to a request it received from the frontend, it will invoke an rpc on the frontend to transmit that response. This PR introduces a second path for this transmission in which the response is streamed instead of sent as one big blob. Which of those two possible options is used can be controlled by the code responsible for computing the response, namely by setting a special header that indicates it wants its response streamed (hence, the response "requests" streaming from the piece of code that decides which path to take).

Maybe it would be less confusing to say `Enables streaming of responses from querier to query-frontend for response types that support it.`?

Reply:
can we also add something in brackets to say that this is only currently supported for the active series API, so we don't give false hope to people expecting improvements on other APIs 😄

Reply:
Changed in 5a56b37c9.